src/components/CustomIcon/CustomIcon.stories.tsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react-native/no-inline-styles */
+import React from 'react';
+import {CustomIcon} from './CustomIcon';
+import {View} from 'react-native';
+import {CustomIconNames} from './CustomIcon.types';
+import {StoryObj} from '@storybook/react-native';
+
+const MyCustomIconMeta = {
+  title: 'MyCustomIcon',
+  component: CustomIcon,
+  args: {
+    name: CustomIconNames.AmazonPay,
+    size: 200,
+  },
+  decorators: [
+    (Story: any) => (
+      <View style={{alignItems: 'center', justifyContent: 'center', flex: 1}}>
```

Comment:
Looks like this decorator is common for all stories. Can add this globally in preview/main? Or do we even need this?